chall05/szavarce.c

```diff
@@ -0,0 +1,42 @@
+int ft_found(char *find, char *in_str)
+{
+    int pos;
+    int posb;
+    int ret;
+
+    pos = 0;
+    posb = 0;
+    ret = 1;
+    while (in_str[pos] != '\0')
+    {
+        posb = 0;
+        ret = 1;
+        while (find[posb])
+        {
+            if (find[posb] != in_str[pos + posb])
```

Comment:
You go out of bounds here for `str = "ie"`. This could cause a crash. You need to check your bounds.